docs/src/main/asciidoc/security-jwt.adoc

```diff
@@ -947,6 +947,7 @@ SmallRye JWT supports the following properties which can be used to customize th
 |smallrye.jwt.new-token.lifespan|300|Token lifespan in seconds which will be used to calculate an `exp` (expiry) claim value if this claim has not already been set.
 |smallrye.jwt.new-token.issuer|none|Token issuer which can be used to set an `iss` (issuer) claim value if this claim has not already been set.
 |smallrye.jwt.new-token.audience|none|Token audience which can be used to set an `aud` (audience) claim value if this claim has not already been set.
+|smallrye.jwt.new-token.override-matching-claims|false| Set this property to `true` for `smallrye.jwt.new-token.issuer` and `smallrye.jwt.new-token.audience` values to override the already initialized `iss` (issuer) and `aud` (audience) claims.
```

Comment:
Can we use the auto-generated Asciidoc config table here?

Reply:
Hi George @gastaldi, can you please link me to some example ? I've used this format as I've seen it somewhere in the docs as well :-)

Reply:
Sure, like this:

https://github.com/quarkusio/quarkus/blob/ba6866fdc747511da7e2f43ad3f069b8e382c16b/docs/src/main/asciidoc/mailer.adoc#L429-L432

Reply:
@gastaldi These are the properties which are coming from `smallrye/smallrye-jwt`, at the moment we have no equivalent `quarkus.smallrye.jwt.*` configuration so I suspect it won't work ?

Reply:
Ah, haven't noticed that. You're right, that makes sense

Reply:
@gastaldi Np :-) this is actually raises the old issue @emmanuelbernard raised awhile back where it would be good to have the quarkus config space only instead of a quarkus/smallrye mix :-), but we'll need to have a dedicated discussion, will start it, thanks :-)